webui/react/src/pages/F_ExpList/glide-table/ColumnPickerMenu.tsx

```diff
@@ -162,6 +163,23 @@ const ColumnPickerTab: React.FC<ColumnTabProps> = ({
     [setSearchString],
   );
 
+  const rows = useCallback(
+    ({ index, style }: { index: number; style: React.CSSProperties }) => {
+      const col = filteredColumns[index];
+      return (
+        <div key={col.column} style={style}>
+          <Checkbox
+            checked={checkedColumn.has(col.column)}
+            id={col.column}
+            onChange={handleColumnChange}>
+            {col.displayName || col.column}
```

Comment:
fix: see if you can't use Typography or something to prevent text wrapping like in this screenshot. otherwise, just apply `overflow-x: hidden` to the div on line 170
![Screenshot 2024-01-02 at 15-23-01 Uncategorized Experiments - Determined](https://github.com/determined-ai/determined/assets/15078396/e5642c8f-7548-4ebb-af41-9bfd1aabfe0d)

Reply:
Great catch, fixed!
One thing I'm not able to reproduce is the blank list you mentioned when switching tabs, I tried to switch as quickly as I can but still not able to reproduce

Reply:
If you go to the Metrics tab, scroll down several popup-heights, switch to the Hyperparameters tab, then switch back to Metrics you should see it. Or at least, I do on Firefox. It seems to be because when you switch tabs the scroll position is (usually? seems like it sometimes doesn't) reset back to the top. Because it's a virtual list the items at the top were unloaded when you scrolled down, so the popup is blank until you start scrolling and it figures out where you are. If you still can't replicate it I can try to record a video.

Reply:
https://www.loom.com/share/074d952f72e348778199c92841757fe8?sid=3ebdc091-c3a7-4bc1-9ede-aa767abe842c

Reply:
Thank you for the video! I can reproduce it locally on Firefox after `make clean & build`, so this means that when my build was with Hew 0.6.22, I could not reproduce the issue. 
So I rebased main and that seems to solve the issue locally and on netlify